connect_box/__init__.py

```diff
@@ -515,29 +519,49 @@ async def _async_initialize_valid_token(self) -> None:
                 timeout=10,
             ) as response:
                 await response.text()
-                #self.token = response.cookies["sessionToken"].value
```

Comment:
On a Swiss Connect Box from UPC (LG-RDK_CH7465LG-NCIP-6.18-2206.7.2-NOSH) `common_page/login.html` give me a 404.

At first glances it looks like that `_async_initialize_valid_token()` is no longer needed.

Reply:
Does the example script work for you when you remove the method? On my `6.15.20.3-GA-NOSH` its still there are needed for the authentication flow

Reply:
6.18-x also has also some endpoints (e.g., `rest/v1/login`).

Reply:
Pushed a fix, which works for my version and hopefully for yours as well. Can you give it another go?